packages/amplify-e2e-core/src/init/amplifyPush.ts

```diff
@@ -36,6 +36,36 @@ export function amplifyPushForce(cwd: string, testingWithLatestCodebase: boolean
   });
 }
 
+/**
+ *
+ * @param cwd
+ * @param stackIdx
+ * @param testingWithLatestCodebase
+ * @returns
+ * Used to stop an iterative deployment
+ */
+export function cancelIterativeAmplifyPush(
+  cwd: string,
+  stackIdx: { current: number; max: number },
+  testingWithLatestCodebase: boolean = false,
+): Promise<void> {
+  return new Promise((resolve, reject) => {
+    spawn(getCLIPath(testingWithLatestCodebase), ['push'], { cwd, stripColors: true, noOutputTimeout: pushTimeoutMS })
+      .wait('Are you sure you want to continue?')
+      .sendLine('y')
+      .wait(`Deploying stack (${stackIdx.current} of ${stackIdx.max})`)
+      .wait(/.*UPDATE_IN_PROGRESS*/)
+      .sendLine(CONTROL_C)
+      .run((err: Error) => {
+        if (err && !/Killed the process as no output receive for | Sec. The no output timeout is set to /.test(err.message)) {
```

Comment:
The regular expression can probably be shortened. Would `Killed the process as no output receive for` be adequate for identifying the error?

Reply:
Shortened phrase